Binary files lite-xl-2.1.1/.meson.build.swp and lite-xl-2.1.1.mod/.meson.build.swp differ
diff -uNr lite-xl-2.1.1/meson.build lite-xl-2.1.1.mod/meson.build
--- lite-xl-2.1.1/meson.build	2022-12-29 10:03:36.000000000 +0800
+++ lite-xl-2.1.1.mod/meson.build	2023-10-03 10:43:59.146852541 +0800
@@ -82,6 +82,7 @@
     # Lua has no official .pc file
     # so distros come up with their own names
     lua_names = [
+        'lua54',
         'lua5.4', # Debian
         'lua-5.4', # FreeBSD
         'lua',    # Fedora
@@ -186,7 +187,6 @@
     lite_bindir = 'bin'
     lite_docdir = 'share/doc/lite-xl'
     lite_datadir = 'share/lite-xl'
-    if host_machine.system() == 'linux'
         install_data('resources/icons/lite-xl.svg',
             install_dir : 'share/icons/hicolor/scalable/apps'
         )
@@ -196,7 +196,6 @@
         install_data('resources/linux/org.lite_xl.lite_xl.appdata.xml',
             install_dir : 'share/metainfo'
         )
-    endif
 endif
 
 install_data('licenses/licenses.md', install_dir : lite_docdir)
